README.md

```diff
@@ -1,16 +1,147 @@
-<table class="table" style="margin-top: 10px">
-    <thead>
-    <tr>
-        <th>Title</th>
-        <th>Last Updated</th>
-        <th>Summary</th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr>
-        <td>Google Cloud Storage Package</td>
-        <td>July 3, 2024</td>
-        <td>This is a package to facilitate interacting with the google cloud storage service for uploading and retrieving files.</td>
-    </tr>
-    </tbody>
-</table>
+
+# Overview
+
+Google Cloud Storage is a cloud-based storage solution that enables secure and reliable storage of data. 
+It allows you to store any amount of data and retrieve it as often as you'd like. 
+The platform provides high availability and offers a variety of storage classes to manage your data based on access frequency. 
+This package integrates with Google Cloud Storage APIs to offer seamless management of your data and buckets.
+
+Repo: [https://github.com/slingr-stack/google-storage-package](https://github.com/slingr-stack/google-storage-package)
+
+This [package](https://platform-docs.slingr.io/dev-reference/data-model-and-logic/packages/) allows you to connect to the Google Cloud Storage API. 
+It provides the following features:
+
+- Authentication via Service Account
+- Direct access to the Google Cloud Storage API
+- Helpers to simplify interactions with Google Cloud Storage in your app
+
+## Configuration
+
+Official documentation: [https://cloud.google.com/storage/docs/json_api](https://cloud.google.com/storage/docs/json_api)
+
+### Registering new service account
+
+To use this package, you must first configure the integration to authenticate with the Google Cloud system using a service account. (On the [Credentials menu](https://console.cloud.google.com/apis/credentials))
+
+You will need to create a service account in your Google Cloud project and download the private key JSON file, which contains the service account email and private key.
+You must format the private key in with characters `\n`!
+
+#### Service account email
+
+**Name**: serviceAccountEmail  
+**Type**: text  
+**Mandatory**: true
+
+#### Private key
+
+**Name**: privateKey  
+**Type**: password (text)  
+**Mandatory**: true
+
+# Javascript API
+
+## HTTP requests
+
+You can make `GET`, `POST`, `PUT`, `PATCH`, and `DELETE` requests to the [Google Cloud Storage API](https://cloud.google.com/storage/docs/json_api) like this:
+
+### List buckets
+
+```javascript
+let response;
+response = pkg.googlecloudstorage.api.get('/b');
+log(JSON.stringify(response));
+```
+
+### Create a bucket
+
+```javascript
+let response;
+response = pkg.googlecloudstorage.api.post('/b', {
+    body: {
+        name: 'my-new-bucket',
+        location: 'US'
+    }
+});
+log(JSON.stringify(response));
+```
+
+### Delete a bucket
+
+```javascript
+let response;
+response = pkg.googlecloudstorage.api.delete('/b/my-bucket');
```

Comment:
we don't support camel case declaration of variable? (`googleCloudStorage`)

Reply:
We support it, but: 
* we have not done extensive coverage on this
* It depends on the first time you register the package in the developer portal, to use camel case in the namespace we have to register the packages correctly and remove all previous versions and metadata (or register the following new packages with this convention)

Reply:
OK, something to keep in mind.